agent/config/types.go

```diff
@@ -359,4 +361,9 @@ type Config struct {
 	// WarmPoolsSupport specifies whether the agent should poll IMDS to check the target lifecycle state for a starting
 	// instance
 	WarmPoolsSupport BooleanDefaultFalse
+
+	// PortRange Variable enables the clients to set their custom port range
```

Comment:
please update this comment block to reflect the var name, something like

```
// DynamicHostPortRange specifies the dynamic host port range that the agent 
// uses to assign host ports for a container port range mapping.
// This defaults to the platform specific ephemeral host port range
```

Reply:
Sure, updated it.